src/EnergyPlus/DataBSDFWindow.hh

```diff
@@ -66,19 +66,31 @@ namespace DataBSDFWindow {
     // Using/Aliasing
     using DataVectorTypes::Vector;
 
-    constexpr int BasisType_WINDOW = 1;
-    constexpr int BasisType_Custom = 2;
+    enum class Basis
+    {
+        Unassigned = -1,
+        WINDOW,
+        Custom
+    };
 
-    constexpr int BasisSymmetry_Axisymmetric = 1;
-    constexpr int BasisSymmetry_None = 2;
+    enum class BasisSymmetry
+    {
+        Unassigned = -1,
+        Axisymmetric,
+        None
+    };
 
     // Thermal calculations for complex fenestration can be used to generate reports for standard cases
     // noCondition is used when performing timestep calculations
     // summerCondtion will override certain parameters so that produced results are matching standard summer WINDOW  = software results
     // winterCondition will override certain parameters so that produced results are matching standard winter WINDOW  = software results
-    constexpr int noCondition = 0;
-    constexpr int summerCondition = 1;
-    constexpr int winterCondition = 2;
+    enum class Condition
+    {
+        Unassigned = -1,
+        noCondition,
+        summerCondition,
```

Comment:
Does `noCondition` also mean `Unassigned` here?

Reply:
I guess so. I'll remove Unassigned. Thanks for your feedback.